### PR TITLE
add meta descriptions to all examples

### LINKED ISSUE #3: add `meta` descriptions to dapp store examples

### PR DESCRIPTION
**Pull request by shadeofblue**
closes #3

### CODE REVIEW

File: apps/http-proxy-specific-port.yaml [hunk 1]
```diff
@@ -1,3 +1,12 @@
+meta:
+  name: "Local HTTP Proxy demo with specific local port"
+  description: "An minimal example presenting a local HTTP proxy connecting to
```
Comment (VanDavv) [inline apps/http-proxy-specific-port.yaml]:
```suggestion
  description: "A minimal example presenting a local HTTP proxy connecting to
```

File: apps/http-proxy.yaml [hunk 2]
```diff
@@ -1,3 +1,11 @@
+meta:
+  name: "Local HTTP Proxy demo"
+  description: "An minimal example presenting a local HTTP proxy connecting to
```
Comment (VanDavv) [inline apps/http-proxy.yaml]:
```suggestion
  description: "A minimal example presenting a local HTTP proxy connecting to
```

**Review by VanDavv — APPROVED** [review]
LGTM, but found typos

File: apps/http-proxy-specific-port.yaml (hunk 1 above)
Comment (shadeofblue) [reply apps/http-proxy-specific-port.yaml]: oops... an artifact of multiple iterations over the content ;)

**Pull request merged**